e2e_deployment_files/long_haul_deployment.template.json

```diff
@@ -279,6 +279,29 @@
               "image": "<Container_Registry>/microsoft/azureiotedge-direct-method-receiver:<Build.BuildNumber>-linux-<Architecture>",
               "createOptions": ""
             }
+          },
+          "moduleRestarter": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "ServiceClientConnectionString": {
+                "value": "<ServiceClientConnectionString>"
```

Comment:
I believe you can parse service client connection string to get device Id, please check.

Reply:
I'm going to use the env var IOTEDGE_DEVICEID. I'm guessing we should make this change to the analyzer as well?